### ISSUE TITLE
With multiple `tel:` links on a page, the first is called when any link is clicked

### ISSUE THREAD

**Issue by nederdirk** [VoIPGRID/firefoxplugin#36]
Example: https://jsfiddle.net/f9r5xLL5/

Given I'm using Firefox with the voipgrid plugin when I click Number 2 then I expect that the voipgrid plugin (which handles tel: links) calls that number.
Actual behavior: the number on the first link is called

**Comment by cpoppema** [comment]
Hi @nederdirk, thanks for reporting this issue!

As you can see by the commit I just posted, this issue is confirmed and I will include it in the next release as fast as possible. However, the current master branch has not been accepted by Mozilla as of yet. For this reason, I also included a new build in the referenced branch. If you cannot wait to use this feature I suggest to download and install the new voipgrid.xpi: https://github.com/VoIPGRID/firefoxplugin/raw/feature/fix-calling-numbers-from-tel-links/voipgrid.xpi.

**Comment by nederdirk** [comment]
Hi @cpoppema, thank you for your quick and adequate reply. I installed the build of the plugin you referenced, will that one be updated if a newer build becomes available via the mozilla plugin site?

**Comment by nederdirk** [comment]
oh, it has the 'Colleagues' feature (which somehow lists phone extensions and not actually my colleagues) too! Thanks!

**Comment by cpoppema** [comment]
Indeed, this feature is new since version 1.6.0 :) The extension for Google Chrome this feature public for some time now as well. The current add-on for Firefox only updates to version 1.5.2 as this is the latest version which has been accepted by Firefox.

I branched from the current master branch which was in review at Mozilla's addon store some time ago. A couple of notes:
* version 1.6.0 has already been rejected by Mozilla,
* version 1.6.1 is currently pending but is not made public until it is accepted. This only concerns small changes regarding the colleagues-tab so it is more stable.

I would not expect Firefox to automatically update an add-on which was installed manually, but I cannot say I have tried, maybe it will work when you'd manually trigger a check for updates. I would advise against updating when you see the aforementioned 1.6.1 since this does not contain the fix for this issue, I assume 1.6.2 will, but I cannot tell when Mozilla will accept this version.
